internal/provider/utils.go

```diff
@@ -41,14 +42,19 @@ func CheckDn(ctx context.Context, client *client.Client, dn string, diags *diag.
 	}
 }
 
-func DoRestRequest(ctx context.Context, diags *diag.Diagnostics, client *client.Client, path, method string, payload *container.Container) *container.Container {
+func DoRestRequest(ctx context.Context, diags *diag.Diagnostics, client *client.Client, path, method string, payload *container.Container, escapeHtml bool) *container.Container {
 
 	// Ensure path starts with a slash to assure signature is created correctly
 	if !strings.HasPrefix("/", path) {
 		path = fmt.Sprintf("/%s", path)
 	}
-
-	restRequest, err := client.MakeRestRequest(method, path, payload, true)
+	var restRequest *http.Request
+	var err error
+	if escapeHtml {
+		restRequest, err = client.MakeRestRequest(method, path, payload, true)
```

Comment:
So for the `standard` resources the escaping is not an issue and we keep same rest request structure?

Reply:
yes, that is right.